nixos/roles/webproxy.nix

```diff
@@ -87,7 +82,12 @@ in
         extraCommandLine = "-s malloc,${toString cacheMemory}M";
         http_address = lib.concatMapStringsSep " -a "
           (addr: "${addr}:8008") fccfg.listenAddresses;
-        config = varnishCfg;
+        virtualHosts = lib.optionalAttrs (varnishCfg != null) {
+          "default-vcl" = {
```

Comment:
This is a bit unpredictable. Could we introduce an order here so that we'd either have this consistently be the first vhost or the last vhost?

Reply:
might try to make it the only vhost with lib.mkForce

Reply:
in fact, the assertion now takes care of that. adding a lib.mkForce here would lead to unintentional behaviour since it would just ignore the user-defined vhosts